README.md

```diff
@@ -1,207 +1,339 @@
+| --geth -\*                           | \* Pass any flag to the Geth node [See docs for details](https://geth.ethereum.org/docs/fundamentals/command-line-options)  |
+| --erigon -\*                         | \* Pass any flag to the Erigon node [See docs for details](https://github.com/ledgerwatch/erigon)                           |
+| --prysm -\*                          | \* Pass any flag to the Prysm node [See docs for details](https://docs.prylabs.network/docs/prysm-usage/parameters)         |
+| --lighthouse -\*                     | \* Pass any flag to the Lighthouse node [See docs for details](https://lighthouse-book.sigmaprime.io/advanced-datadir.html) |
+| --validator -data-dir                | A path of validator's data directory (./validator_data)                                                                     |
+| --validator -verbosity               | verbosity for validator logs                                                                                                |
+| --validator -wallet-dir              | location of a generated wallet (./mainnet_keystore)                                                                         |
+| --validator -wallet-password-file    | location of password used for wallet generation (./config/mainnet/shared/secrets/validator-password.txt)                    |
+| --validator -chain-config-file       | path to config.yaml file (./config/mainnet/shared/config.yaml)                                                              |
+| --validator -monitor-host            | host used for interacting with Prometheus metrics (IP address)                                                              |
+| --validator -grpc-gateway-host       | host for gRPC gateway (IP address)                                                                                          |
+| --validator -rpc-host                | RPC server host (IP address)                                                                                                |
+| --validator -suggested-fee-recipient | address that receives block fees (0x12345..abcd)                                                                            |
+| --validator -output-dir              | directory where logs are created                                                                                            |
+| --validator -std-output              | set output to console                                                                                                       |
+| --log -folder "./myCustomLogFolder"  | user can setup a custom log directory when starting LUKSO client                                                            |
+| --log -size n                        | Log files capped to the size in MB                                                                                          |
```

Comment:
Sizing feature is not implemented for now and has low priority, so I think we can omit this one